### PR TITLE
N7s aur#237 datastore test batch 2

### PR DESCRIPTION
**Pull request by Akoszewski**

### CODE REVIEW

File: requirements/AUR_CBI_MDL-0120/testing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+**1. Steps:**
+
+* Build the project with "make" command
+* Run ./work/binaries/demo file
+
+**2. Expected results:**
+
+* Project should succesfully build
+* The running binary should output logs in a loop with the line: Custom value of custom type: 42
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0120/testing.md]: "[AUR-CBI-MDL-0120]The type of the DataItems stored by the DataStore component shall be user configurable" - while in other tests it is preferred to link to the reference ASN.1, in this case, I would prefer to see ASN.1 with altered (different than in all other tests) DataItem type.

File: requirements/AUR_CBI_MDL-0130/testing.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+**2. Expected results:**
+
+* Project should succesfully build
+* The running binary should output logs in a loop with the following lines at the beginning:
+
+    client1: item created 1
+    client1: item created 2
+    client1: item created 3
+    client1: item removed 3
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0130/testing.md]: "[AUR-CBI-MDL-0130]The DataStore component shall have the capability to automatically remove the oldest DataItem when it is full and receiving a new DataItem"
Shouldn't this be item removed 1?

Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0130/testing.md]: That's right, I just copied the actual output. Looks like there is an error in the current implementation. I will change the expected output to what it should be.

File: requirements/AUR_CBI_MDL-0120/testing.md (hunk 1 above)
Comment (Akoszewski) [reply requirements/AUR_CBI_MDL-0120/testing.md]: OK

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**